src/views/Login/Ledger.js

```diff
@@ -78,7 +78,7 @@ export default function Ledger({ className, goHome }) {
           walletHdPath: addHdPrefix(values.hdpath),
         });
 
-        setAuthToken(authToken);
+        setAuthToken(Just(authToken));
```

Comment:
I can't remember why this is a `Maybe`. Don't we have this in all post-login cases?  
(Feel like I asked this already at some point, but didn't take notes well enough. Sorry!)

Reply:
It was more for consistency with the rest of useWallet